example/windows/ProgressViewExample/App.cpp

```diff
@@ -0,0 +1,42 @@
+#include "pch.h"
+
+#include "App.h"
+
+#include "AutolinkedNativeModules.g.h"
+#include "ReactPackageProvider.h"
+
+
+using namespace winrt::ProgressViewExample;
+using namespace winrt::ProgressViewExample::implementation;
+
+/// <summary>
+/// Initializes the singleton application object.  This is the first line of
+/// authored code executed, and as such is the logical equivalent of main() or
+/// WinMain().
+/// </summary>
+App::App() noexcept
+{
+     MainComponentName(L"ProgressViewExample");
+
+#if BUNDLE
+    JavaScriptBundleFile(L"index.windows");
+    InstanceSettings().UseWebDebugger(false);
+    InstanceSettings().UseFastRefresh(false);
+#else
+    JavaScriptMainModuleName(L"index");
+    InstanceSettings().UseWebDebugger(true);
+    InstanceSettings().UseFastRefresh(true);
+#endif
+
+#if _DEBUG
+    InstanceSettings().EnableDeveloperMenu(true);
+#else
+    InstanceSettings().EnableDeveloperMenu(false);
+#endif
+
+    RegisterAutolinkedNativeModulePackages(PackageProviders()); // Includes any autolinked modules
+    PackageProviders().Append(winrt::progress_view::ReactPackageProvider());
+    PackageProviders().Append(make<ReactPackageProvider>()); // Includes all modules in this project
+
+    InitializeComponent();
+}
```

Comment:
I think this might be using an older template? it doesn't have a main page to navigate to